third_party/blink/renderer/core/html/media/html_media_element.cc

```diff
@@ -418,8 +419,8 @@ bool IsProgressiveFormat(const ContentType& content_type) {
   }
 
   Vector<String> split_codecs;
-  const String separator(", ");
-  codecs.Split(separator, split_codecs);
+  const String separator(",");
+  codecs.StripWhiteSpace().Split(separator, split_codecs);
```

Comment:
StripWhiteSpace() seems to only remove white spaces from the front and back, so the current implementation won't resolve the comment in the previous PR.  Since we only check whether there are two codecs, we don't have to strip white spaces here.

Also, consider updating the test case as Miguel suggested.

Reply:
I've replaced StripWhiteSpace() with RemoveCharacters(), which I tested removes whitespace within the string as intended.

Reply:
I would suggest remove the call to `RemoveCharacters()` here.  It doesn't improve (it actually reduces) the accuracy of the implementation and the new test case passes with or without it.

A proper implementation is to strip after split, which isn't necessary here as we only check `split_codecs.size() == 2`.

Reply:
I'll remove it